securedrop/manage.py

```diff
@@ -18,8 +19,11 @@
 from db import db_session, init_db, Journalist
 from management.run import run
 
+logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s')
+log = logging.getLogger(__name__)
 
-def reset():  # pragma: no cover
+
+def reset(args):  # pragma: no cover
```

Comment:
Why did you add `args` to so many functions when it does not appear to be used in any of them?